### PR TITLE
Add `login-username` support

### PR DESCRIPTION
**Pull request by Mstrodl**
Shown for the "Username Form" login step.

I would also like to work on:
- `login-password.ftl`
- `webauthn-authenticate.ftl`

But first want to see opinions on this!

p.s. I would appreciate a `hacktoberfest-accepted` label if you're okay with it!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
         "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
     },
     "devDependencies": {
+        "@babel/core": "^7.0.0",
```
Comment (Mstrodl) [inline package.json:62]: I was getting peer dependency errors from this. Let me know if this is an issue

File: src/bin/keycloakify/generateFtl/generateFtl.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import { Reflect } from "tsafe/Reflect";
 // https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/forms/login/freemarker/Templates.java
 export const pageIds = [
     "login.ftl",
+    "login-username.ftl",
```
Comment (Mstrodl) [inline src/bin/keycloakify/generateFtl/generateFtl.ts:16]: These didn't appear in any particular order. Is this an okay place to put this?

Comment (garronej) [reply src/bin/keycloakify/generateFtl/generateFtl.ts:16]: sure

File: package.json (hunk 1 above)
Comment (garronej) [reply package.json:62]: No problem

**Comment by garronej** [comment]
Hey @Mstrodl thank you so much for this great contribution.  

Would you run `yarn format`?

**Comment by Mstrodl** [comment]
> Hey @Mstrodl thank you so much for this great contribution.
> 
> Would you run `yarn format`?

Hmm, I did try this. Looks like I forgot to regenerate the lockfile (I use `pnpm` normally). Should be fixed now!

**Comment by garronej** [comment]
@Mstrodl I've [updated the log](https://github.com/InseeFrLab/keycloakify/blob/5fa09152711c958c48455475a1f00618cf0fe3f1/README.md?plain=1#L54-L56) and released a new version `6.5.0`.  

Thanks again! 

**Comment by garronej** [comment]
@all-contributors please add @Mstrodl for code

**Pull request merged**